src/__experimental__/components/decimal/decimal.component.js

```diff
@@ -247,7 +247,7 @@ class Decimal extends React.Component {
   removeDelimiters = (value) => {
     const format = I18nHelper.format();
     const delimiter = `\\${format.delimiter}`;
-    const delimiterMatcher = new RegExp(`[${delimiter}]*`, "g");
+    const delimiterMatcher = new RegExp(`[${delimiter}^´¨˙~]*`, "g");
```

Comment:
The intention of `removeDelimiters` was purely to remove the delimiter as provided by I18n.

(in UKI that would be a `,`, so it would convert `10,000.00` down to `10000.00`)

not sure we should use this to also match other characters like `^´¨˙~`